src/System.Management.Automation/engine/NativeCommandParameterBinder.cs

```diff
@@ -196,62 +196,83 @@ private void AppendOneNativeArgument(ExecutionContext context, object obj, Array
                     }
                 }
 
-                if (!string.IsNullOrEmpty(arg))
+                if (!ExperimentalFeature.IsEnabled("PSNativePSPathResolution") &&
+                    arg.Length == 0)
```

Comment:
Do we need a null check for arg?

Reply:
From a type perspective, I don't see how it can ever be null. I think it either works or throws.